react/components/atoms/container-swipeable-horizontal/container-swipeable-horizontal.stories.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { View, StyleSheet } from "react-native";
+import { withKnobs, number, text, boolean, color, select } from "@storybook/addon-knobs";
+
+import { ContainerSwipeableHorizontal } from "../..";
+
+storiesOf("Atoms", module)
+    .addDecorator(withKnobs)
+    .add("Container Swipeable Horizontal", () => {
+        const triggerActionThreshold = number("Trigger Action Threshold Value", 0.4);
+        const leftOptionEnabled = boolean("Left Option Enabled", false);
+        const rightOptionEnabled = boolean("Right Option Enabled", false);

```

Comment:
Set one of this options to _true_. Because its a little bit weird entering the storie in storybook and have to activate the prop to see the component working